sn_client/src/wallet.rs

```diff
@@ -127,7 +126,33 @@ impl WalletClient {
         self.wallet.unconfirmed_spend_requests()
     }
 
-    /// Get the payment for a given network address
+    /// Get the payment detail from a given address within the network.
```

Comment:
I think this function returns the cached `Payment` for the provided `NetworkAddress`. I.e., we pay for addresses before we can store anything at that location, and this retrieves the payment that the client has made.